src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToLocalizedStringTransformer.php

```diff
@@ -199,4 +200,12 @@ protected function isPatternDateOnly()
         // check for the absence of time-related placeholders
         return 0 === preg_match('#[ahHkKmsSAzZOvVxX]#', $pattern);
     }
+
+    private function getConverterFormatter()
+    {
+        // see https://bugs.php.net/bug.php?id=66323
+        $timezone = class_exists('IntlTimeZone', false) ? \IntlTimeZone::createTimeZone('UTC') : 'UTC';
```

Comment:
we can already past timezone as a string :/ the bug report is for using integers (e.g. derived from DateTimeZone::EUROPE) which is not supported here.